### PR TITLE
Data : Pravesh-Sudha

### PR DESCRIPTION
**Pull request by Pravesh-Sudha**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7734"></a>

### CODE REVIEW

**Review by Ananya2001-an — APPROVED** [review]
Looks good!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Review by thisisdishang — APPROVED** [review]
Kindly update the title example "data: your github username"

**Review by accodes21 — COMMENTED** [review]
Resolve the conflict by making your fork up to date

**Comment by SaraJaoude** [comment]
> Resolve the conflict by making your fork up to date

@accodes21 please avoid duplicating comments - this adds extra unnecessary notifications and can confuse the author

File: data/Pravesh-Sudha.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Pravesh-Sudha",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/pravesh-sudha-82a37422a/",
+      "icon": "FaLinkedIn"
```
Comment (RohanSasne) [inline data/Pravesh-Sudha.json:26]:
```suggestion
      "icon": "FaLinkedin"
```

Symbols are case sensitive :)

**Review by RohanSasne — CHANGES_REQUESTED** [review]

**Pull request closed without merging**